### PR TITLE
chore(package.json): update packge scripts

### PR DESCRIPTION
**Pull request by kwonoj**
- exclude newly added helper from code coverage
- bump up closuure, commitizen dependency version
- update wallaby configuration

set of minor, minor changes to enhance ergonomics slightly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,15 +18,15 @@
     "build_es6": "rm -rf dist/es6 && tsc src/Rx.ts src/Rx.KitchenSink.ts --outDir dist/es6 --sourceMap --target ES6 -d --diagnostics",
     "build_closure": "java -jar ./node_modules/google-closure-compiler/compiler.jar ./dist/global/Rx.js --create_source_map ./dist/global/Rx.min.js.map --js_output_file ./dist/global/Rx.min.js",
     "build_global": "rm -rf dist/global && mkdir \"dist/global\" && browserify src/Rx.global.js --outfile dist/global/Rx.js && npm run build_closure",
-    "build_perf": "npm run build_es6 && npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
-    "build_test": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && jasmine",
-    "build_cover": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && npm run cover",
+    "build_perf": "npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
+    "build_test": "rm -rf dist/ && npm run lint && npm run build_cjs && jasmine",
```
Comment (kwonoj) [inline package.json]: Now cjs does not depends on es6 build results, removed some of tasks has chained build jobs for fast turnaround on dev machines. Travis still runs build_all.

File: src/observables/ForkJoinObservable.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export class ForkJoinObservable<T> extends Observable<T> {
                                     Promise<any> |
                                     ((...values: Array<any>) => any)>) {
      super();
-   }
+  }
```
Comment (kwonoj) [inline src/observables/ForkJoinObservable.ts]: credit to @staltz , hawkeye could catch my indentation error. :heart_eyes:

**Comment by kwonoj** [comment]
I can check in this myself, but will do after got agreement on `build_test` (and some other) behavior changes to skip es6 build.

**Comment by blesh** [comment]
> I can check in this myself, but will do after got agreement on build_test (and some other) behavior changes to skip es6 build.

That's fine. I agree, there's no reason to do the es6 build first.

**Comment by kwonoj** [comment]
Merged with 92a247d .

File: src/observables/ForkJoinObservable.ts (hunk 2 above)
Comment (staltz) [inline src/observables/ForkJoinObservable.ts]: lol

Comment (staltz) [inline src/observables/ForkJoinObservable.ts]: What if I'd say now line 11 has wrong indentation, since `super()` is 3 spaces to the right? (1 too much)

Comment (kwonoj) [inline src/observables/ForkJoinObservable.ts]: When @staltz reads code : 



I'll get it updated :)

**Pull request closed without merging**